### PR TITLE
Edits to files-part 2

### PR DESCRIPTION
**Pull request by PipKat**
Edits to ensure that all doc and coding style content removed from the Ansys Sphinx Template documentation and the PyAEDT documentation exists somewhere in the PyAnsys Developer's Guide.

### CODE REVIEW

**Comment by PipKat** [comment]
I can see how specifying a reference designator is better than using the title. I'm sure there there is a better way to catch problems than letting the Documentation check identify them one by one, but I'm not sure what that way is.

File: doc/source/how-to/testing.rst [hunk 1]
```diff
@@ -510,7 +510,7 @@ Using a ``codecov.yml`` file requires that each PR has a patch coverage of 90%,
 source added to the repository (unless ignored) must be covered by unit tests.
 
 
-Testing using GitHub Actions
+Test using GitHub Actions
 ----------------------------
```
Comment (RobPasMue) [inline doc/source/how-to/testing.rst:514]:
```suggestion
Test using GitHub Actions
-------------------------
```

File: doc/source/how-to/testing.rst [hunk 2]
```diff
@@ -433,13 +433,13 @@ the `pytest-cov <https://pytest-cov.readthedocs.io/>`_ library to view the
 coverage for your library.
 
 
-Configuring code coverage
+Configure code coverage
 ~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (RobPasMue) [inline doc/source/how-to/testing.rst:437]:
```suggestion
Configure code coverage
~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/source/how-to/testing.rst [hunk 3]
```diff
@@ -423,7 +423,7 @@ and integration testing is complete. Ideally, you should run them outside the
 
 
 
-Testing code coverage
+Test code coverage
 ---------------------
```
Comment (RobPasMue) [inline doc/source/how-to/testing.rst:427]:
```suggestion
Test code coverage
------------------
```

File: doc/source/how-to/releasing.rst [hunk 4]
```diff
@@ -366,43 +363,42 @@ You can download a Python package from `PyAnsys PyPI`_ by running:
...
-obtain your unique project token, contact `Alexander Kaszynski
-<mailto:alexander.kaszynski@ansys.com>`_ or `Maxime Rey
-<mailto:maxime.rey@ansys.com>`_. Once you have received it,
+obtain your unique project token, email
+`pyansys.support@ansys.com <pyansys.support@ansys.com>`_. Once you have received it,
 update the token value in the ``Secrets/Actions`` section.
 
-Publishing to GitHub
+Publish to GitHub
 ~~~~~~~~~~~~~~~~~~~~
```
Comment (RobPasMue) [inline doc/source/how-to/releasing.rst:394]:
```suggestion
Publish to GitHub
~~~~~~~~~~~~~~~~~
```

File: doc/source/how-to/releasing.rst [hunk 5]
```diff
@@ -39,12 +36,12 @@ to be much more stable.
...
     For example, the PyMAPDL library ``ansys-mapdl-core`` might have the version
-    ``0.59.0`` whereas the product version is 21.2 (2021 R2). The reason behind
+    ``0.59.0`` whereas the product version is 22.2 (2022 R2). The reason behind
     this is PyAnsys libraries are expected to be developed outside the product
     release cycle in a rapid CI/CD manner.
 
-Branching model
----------------
+Branch model
+-------------
```
Comment (RobPasMue) [inline doc/source/how-to/releasing.rst:44]:
```suggestion
Branch model
------------
```

File: doc/source/how-to/contributing.rst [hunk 6]
```diff
@@ -211,7 +211,7 @@ visible in the branch that you just pushed.
 
    git push -u origin <new branch name>
 
-Creating a pull request
+Create a pull request
 -----------------------
```
Comment (RobPasMue) [inline doc/source/how-to/contributing.rst:215]:
```suggestion
Create a pull request
---------------------
```

File: doc/source/how-to/contributing.rst [hunk 7]
```diff
@@ -41,12 +41,12 @@ boards.
      </div>
 
 
-Creating a GitHub account
+Create a GitHub account
 -------------------------
```
Comment (RobPasMue) [inline doc/source/how-to/contributing.rst:45]:
```suggestion
Create a GitHub account
-----------------------
```

**Review by RobPasMue — APPROVED** [review]
LGTM! Only some minor comments. Thanks @PipKat

File: doc/source/doc-style/index.rst [hunk 8]
```diff
@@ -38,14 +38,16 @@ follow the guidelines in the `Google developer documentation style guide
...
+information, see :ref:`Vale`.
 
 Finally, the documentation should be public and hosted via gh-pages, either as
 a branch named ``gh-pages`` within the library repository or within a
-``gh-pages`` branch within ``<library-repository>-docs``. For more information,
-see :ref:`Deploying documentation`.
+``gh-pages`` branch within ``<library-repository>-docs``. 
+
+For procedural information related to crafting, building, and deploying
+documentation, see :ref:`Documenting` in the How-to section.
```
Comment (jorgepiloto) [inline doc/source/doc-style/index.rst:50]: Maybe, we could also link the `How-to` section by doing:

```suggestion
documentation, see :ref:`Documenting` in the :ref`How-to` section.
```

File: doc/source/how-to/releasing.rst [hunk 9]
```diff
@@ -184,11 +181,11 @@ Documentation in the form of PDF and HTML files are also considered artifacts.
 
 Manual release processes are discouraged and should be avoided. Automated
 release processes minimize human-error and alleviate workload on teams. For more
-information, see :ref: `Automating Release Process`.
+information, see :ref: `Automate release process`.
```
Comment (jorgepiloto) [inline doc/source/how-to/releasing.rst:184]:
```suggestion
information, see :ref:`Automate release process`.
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
Hi @PipKat,
thanks for addressing this. @RobPasMue spotted most of the required changes before merging this, thank you Roberto. I just added a couple of suggestions.

Glad to see our docs improve every day! 🚀

File: README.rst [hunk 10]
```diff
@@ -11,12 +11,13 @@ document for:
 - Anyone who is interested in learning more about the PyAnsys 
   project and PyAnsys libraries
 
-A link to the latest web-based version of this guide is available in the **About**
-area of the repository. In the `Releases
-<https://github.com/pyansys/about/releases>`_ area of the repository, you can
-view information about all releases of this guide. In the **Assets**  area for
-any release, you can download a PDF of the guide for this release.
+This guide describes the PyAnsys project organization and administration.
+It also provides how-to information and summarizes packing, coding, and
```
Comment (MaxJPRey) [inline README.rst:15]:
```suggestion
It also provides how-to information and summarizes packaging, coding, and
```

File: README.rst [hunk 11]
```diff
@@ -11,12 +11,13 @@ document for:
...
+documentation styles.It concludes with an abstractions section that explains
+how apps and complex services expose functionalities that matter to users and
+hide all else.
 
-This guide provides an overview that describes the PyAnsys project organization,
-project administration, and how to contribute. It also provides guidelines and best
-practices, and it summarizes coding and documentation styles.
+From the **About** area for the repository, you can click the for the latest web-based
```
Comment (MaxJPRey) [inline README.rst:20]:
```suggestion
From the **About** area for the repository, you can click for the latest web-based
```

File: doc/source/how-to/releasing.rst [hunk 12]
```diff
@@ -184,11 +181,11 @@ Documentation in the form of PDF and HTML files are also considered artifacts.
...
 
 
-Publishing privately on PyPI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Publish privately on PyPI
+~~~~~~~~~~~~~~~~~~~~~~~~~
 It is sometimes necessary to host and pull packages that are not ready to be
 hosted on the public `PyPI`_. For example, if a PyAnsys library requires
 auto-generated gRPC' interface files from a feature or service that is still
```
Comment (MaxJPRey) [inline doc/source/how-to/releasing.rst:191]:
```suggestion
auto-generated gRPC interface files from a feature or service that is still
```

**Review by MaxJPRey — APPROVED** [review]

File: doc/source/how-to/releasing.rst [hunk 13]
```diff
@@ -184,14 +181,14 @@ Documentation in the form of PDF and HTML files are also considered artifacts.
 
 Manual release processes are discouraged and should be avoided. Automated
 release processes minimize human-error and alleviate workload on teams. For more
-information, see :ref: `Automating Release Process`.
+information, see :ref:`Automate release process`.
```
Comment (RobPasMue) [inline doc/source/how-to/releasing.rst:184]:
```suggestion
information, see :ref:`Automate the release process`.
```

Title changed name =)

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**